draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -475,9 +475,157 @@ Additional JWT VC Issuer Metadata configuration parameters MAY also be used.
+
+ * Developers of Issuers and Verifiers can use the Typ Metadata to understand the
+   semantics of the type and the associated rules. While in some cases,
+   Issuers are the parties that define types (credential formats), this is
+   not always the case. For example, a type can be defined by a
+   standardization body or a community.
+ * Verifiers can use the metadata to determine whether a credential is valid
+   according to the rules of the type. For example, a Verifier can check
+   whether a credential contains all required claims and whether the claims
+   are selectively disclosable.
+ * Wallets can use the metadata to display the credential in a way that is
+   consistent with the Issuer's intent.
+
+Applications using Type Metadata defined in this specification are called "Consumers" in the following. This typically includes Issuers, Verifiers, and Wallets.
+
+## Type Metadata Example {#type-metadata-example}
+
+All examples in this section are non-normative.
+
+Type Metadata about the value of a `vct` claim value can be retrieved as described in (#retrieving-type-metadata).
+
+```json
+{
+  "vct": "https://betelgeuse.example.com/education_credential",
+  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
+  ...
+}
+```
+
+Here, the type is `https://betelgeuse.example.com/education_credential`. The
+Type Metadata is retrieved from the URL
+`https://betelgeuse.example.com/.well-known/vct/education_credential`.
+
+The following is an example for a Type Metadata document:
+
+```json
+{
+  "vct":"https://betelgeuse.example.com/education_credential",
+  "name":"Betelgeuse Education Credential - Preliminary Version",
+  "description":"This is our development version of the education credential. Don't panic.",
+  "extends":"https://galaxy.example.com/galactic-education-credential-0.9",
+  "extends#integrity":"sha256-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-WRL5"
+}
+```
+
+## Type Metadata Format {#type-metadata-format}
+
+The Type Metadata document MUST be a JSON object. The following properties are
+defined:
+
+* `name`
+  * OPTIONAL. A human-readable name for the type, intended for developers reading
+  the JSON document.
+* `description`
+  * OPTIONAL. A human-readable description for the type, intended for
+  developers reading the JSON document.
+* `extends`
+  * OPTIONAL. A URI of another type that this type extends, as described in
+  (#extending-type-metadata).
+
+## Extending Type Metadata {#extending-type-metadata}
+
+A type can extend another type. The extended type is identified by the URI in
+the `extends` property. Consumers MUST retrieve and process
+Type Metadata for the extended type before processing the Type Metadata for the extending
+type.
+
+The extended type MAY itself extend another type. This can be used to create a
+chain or hierarchy of types. The security considerations described in
+(#circular-extends) apply in order to avoid problems with circular dependencies.
```

Comment:
I do believe we need a bit more text here on how exactly extending works:
- Can we only introduce new claims and re-use the parts we are extending (which is what I would assume so far?)
- Can we also override existing information?

Reply:
Indeed, but that should be defined in the context (and the PR) of the particular property, e.g., the display property or the schema.